packages/visual-editor/src/internal/hooks/layout/useMessageReceivers.ts

```diff
@@ -87,12 +87,21 @@ export const useLayoutMessageReceivers = (
       path = resolvePageSetUrlTemplate(streamDocument, "");
     }
 
+    // Find the relativePrefixToRoot for the page being rendered in the editor
+    const pathComponents = path.split("/");
+    pathComponents.pop();
+    const relativePrefixToRoot = pathComponents
```

Comment:
relativePrefixToRoot should be available already in the TemplateProps. Does it need to be different for the editor's sake?